project.godot

```diff
@@ -71,7 +71,7 @@ window/size/height=540
 
 [gdnative]
 
-singletons=[ "res://addons/godot_ovrmobile/godot_ovrmobile.gdnlib" ]
+singletons=[ "res://addons/godot-openvr/godot_openvr.gdnlib", "res://addons/godot_ovrmobile/godot_ovrmobile.gdnlib" ]
```

Comment:
can you remove the dependency on openvr here? Not sure what will happen when someone does not have the openvr module installed.

Reply:
The asset was loaded to test out OpenVR, one of the interfaces supported by the library.  I've not looked into the project file before, but when you disable it you merely get the new line: 

> singletons_disabled=[ "res://addons/godot_ovrmobile/godot_ovrmobile.gdnlib" ]

In order to remove that invisible dependency, I have to delete the whole addons/godot_ovrmobile directory, and then the project file updated what it knew was on the disk.  So, technically, I don't think it makes a difference what is in here because it updates it on load.  But I've deleted it.

Reply:
ah; ok; thanks; I was not sure what happens when a user does not have openvr installed. But if it does not throw an error it is fine.